tests/buildenv.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -eu
```

Comment:
run test scripts with set -eux by default. it is more verbose, but they are tests after all and improves debug ability (especially on the CI console but even so locally)